src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/how-to-upgrade-from-lts-2023-to-lts-2025/how-to-upgrade-search-service.md

```diff
@@ -0,0 +1,309 @@
+---
+title: How to upgrade to New Search Service
+description: Instructions to upgrade your Nuxeo Project to LTS 2025 - Nuxeo Search Service Part
+review:
+   comment: ''
+   date: '2025-04-21'
+   status: ok
+labels:
+    - release-notes
+toc: true
+tree_item_index: 250
+---
+
+## Introduction
+
+{{! excerpt}}
+A new `SearchService` has been introduced in Nuxeo LTS 2025, the `ElasticSearchService` has been removed, representing a breaking change. The code needs to be modified to use the new `SearchService`.
+{{! /excerpt}}
+
+## Maven Dependency
+
+`nuxeo-elasticsearch-core` has been removed in LTS 2025
+```xml
+<dependency>
+  <groupId>org.nuxeo.elasticsearch</groupId>
+  <artifactId>nuxeo-elasticsearch-core</artifactId>
+</dependency>
+```
+Should be replaced by:
+```xml
+<dependency>
+  <groupId>org.nuxeo.ecm.core</groupId>
+  <artifactId>nuxeo-core-search</artifactId>
+</dependency>
+```
+
+The management API provided by `nuxeo-elasticsearch-rest` is now part of `nuxeo-rest-api-server`.
+
+The management API supersedes the old `Elasticsearch.*` automation operation to reindex.
```

Comment:
```suggestion
The management API supersedes the old `Elasticsearch.*` automation operations to reindex.
```
In `Elasticsearch.*`, the `*` means many operations?

Reply:
yes I will rephrase.